src/shared/components/Pagination/Pagination.style.tsx

```diff
@@ -1,50 +1,67 @@
-import { colors, sizes, transitions, typography } from '@/shared/theme'
+import { breakpoints, colors, sizes, transitions, typography } from '@/shared/theme'
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
+import Button from '../Button'
 
 type PaginationButtonProps = {
-  isChevron?: boolean
   isActive?: boolean
   isHidden?: boolean
 }
 
 export const PaginationWrapper = styled.div`
   display: flex;
+  max-width: 400px;
+  justify-content: space-between;
+  @media screen and (min-width: ${breakpoints.small}) {
+    max-width: unset;
+    justify-content: unset;
+  }
 `
 
 const buttonActiveState = css`
   color: ${colors.white};
   background-color: ${colors.gray[800]};
 `
 
+export const ChevronButton = styled(Button)`
+  width: ${sizes(12)};
+  height: ${sizes(12)};
+  @media screen and (min-width: ${breakpoints.small}) {
+    &:first-of-type {
+      margin-right: ${sizes(8)};
+    }
+    &:last-of-type {
+      margin-left: ${sizes(8)};
+    }
+  }
+`
+
 export const PaginationButton = styled.button<PaginationButtonProps>`
```

Comment:
The background-color we have don't seem to be matching what's in Figma. We should use those: https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=6530%3A2651

First for inactive item, second for hover, third for active state (so both current page and pressed button)

Reply:
Everything looks fine on Figma where we have a grayish background, but if we set these transparent colors on the black background - active buttons will be barely visible at least for my taste. I'll update this, but want to just raise the issue.
![image](https://user-images.githubusercontent.com/51168865/114003558-216be900-985e-11eb-85ae-53835fa745ba.png)

Reply:
Good point, I'll raise this with the designers. For now, let's use what's in the design